jetty-runner/src/test/java/org/eclipse/jetty/maven/jettyrunner/it/IntegrationTestJettyRunner.java

```diff
@@ -31,13 +31,12 @@
     @Test
     public void testGet() throws Exception
     {
-        int port = findPortUsed();
+        String serverUri = findServerUri();
         HttpClient httpClient = new HttpClient();
         try
         {
             httpClient.start();
-            ContentResponse response =
-                httpClient.newRequest("localhost", port).path("/").send();
+            ContentResponse response = httpClient.newRequest(serverUri).send();
```

Comment:
Keep in mind that the serverUri will contain the contextPath of the first ContextHandler, if it isn't `/` then you'll need to `URI.create(serverUri).resolve("/")` to use the path at root.

Reply:
but here we have only one war deployed.

Reply:
interesting the documentation says `[[--path /path] context]*n` but looking at the code we do not support this `n` ;)